.gitignore

```diff
@@ -5,3 +5,4 @@
 Manifest
 pkg
 .bundle
+.project
```

Comment:
@joelvh This should ideally instead be in your user ```~/.gitconfig``` as it has nothing to do with this project.